_data/services.yml

```diff
@@ -10,8 +10,8 @@ docs:
 - title: Coaching
   url: /coaching/
```

Comment:
One thing to update, is to change these URLs to not have the trailing slash, so this would just be /coaching

Sorry I missed these in all the updates and shuffling of this stuff.

Reply:
Can you accept this PR and I'll add it to the one related to cleaning up the categories I'm doing right now?

Reply:
yup